src/index.html

```diff
@@ -1,6 +1,18 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Remember to use 2 spaces for indentation in your file. The indentation here seems to be more than 2 spaces. Also, ensure that lines of code have 80 characters max for better readability. The `href` attribute value in the `link` tag is quite long and should be broken into multiple lines.